Editor/Core/Config/ConfigureGame.cs

```diff
@@ -103,17 +103,27 @@ private static void GetReferences(string packageName, ThunderKitSettings setting
         {
             Debug.Log("Acquiring references");
             var blackList = AppDomain.CurrentDomain.GetAssemblies()
-                .Where(asm => !asm.IsDynamic)
-                .Select(asm => Path.GetFileName(asm.Location))
+                //.Where(asm => !asm.IsDynamic)
```

Comment:
IsDynamic was commented out to get the code compiling for NET 3.5 however it was implemented to avoid cases that would throw exceptions.
The specific reasons are lost in time and this should be reviewed to determine what impact this change has

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.isdynamic?view=net-5.0

Reply:
https://stackoverflow.com/questions/1423733/how-to-tell-if-a-net-assembly-is-dynamic

Reply:
This is dealing with situations where runtime generated assemblies are present.  support needs to be maintained here or a less aggressive acquisition of assemblies needs to be used.
The StackOverflow article suggests some alternatives that may work in place of IsDynamic in 3.5